client/src/components/Form/Form.js

```diff
@@ -1,38 +1,52 @@
-import React, { Component } from 'react';
+import React, { useEffect, useRef } from 'react';
 import FormAlert from 'components/FormAlert/FormAlert';
 import PropTypes from 'prop-types';
 
-class Form extends Component {
-  constructor(props) {
-    super(props);
-    this.handleSubmit = this.handleSubmit.bind(this);
-  }
+const Form = ({
```

Comment:
The following props were removed - defined in propTypes but not used in code:
- extraClass